hack/versions/cmd/schema_check/check.go

```diff
@@ -17,13 +17,13 @@ limitations under the License.
 package main
 
 import (
-	"github.com/sirupsen/logrus"
+	"os"
 
 	"github.com/GoogleContainerTools/skaffold/hack/versions/pkg/schema"
 )
 
 func main() {
 	if err := schema.RunSchemaCheckOnChangedFiles(); err != nil {
-		logrus.Fatal(err)
+		os.Exit(1)
```

Comment:
This doesn't seem right since errors reading config files (lines 50-80) will be silently dropped.